github-electron/github-electron.d.ts

```diff
@@ -1186,6 +1186,11 @@ declare module Electron {
 		 * before using this API
 		 */
 		checkForUpdates(): any;
+		/**
+		 * Restarts the app and installs the update after it has been downloaded.
+		 * It should only be called after update-downloaded has been emitted.
+		 */
+		 quitAndInstall(): any;
```

Comment:
`quitAndInstall()` looks to return nothing.  Could you change return type to `void`?

https://github.com/atom/electron/blob/52f1431bde557e083f74af368aed5c08b9ae76f1/atom/browser/api/atom_api_auto_updater.cc#L89